tests/lib/eslint/eslint.js

```diff
@@ -158,6 +159,10 @@ describe("ESLint", () => {
                 assert.strictEqual(ESLint.configType, "flat");
             });
 
+            it("should have the defaultConfig static property", () => {
+                assert.deepStrictEqual([...ESLint.defaultConfig], defaultConfig);
```

Comment:
```suggestion
                assert.deepStrictEqual(ESLint.defaultConfig, defaultConfig);
```
how does spreading it affect the comparison with `defaultConfig`?

Reply:
No, it doesn't.